src/models/CategoryModel.ts

```diff
@@ -1,19 +1,23 @@
-// @flow
-
-import type Moment from 'moment'
+import { Moment} from 'moment'
 import CategoriesMapModel from './CategoriesMapModel'
 import ExtendedPageModel from './ExtendedPageModel'
 
 class CategoryModel extends ExtendedPageModel {
+
   _root: boolean
   _parentPath: string
   _order: number
 
-  constructor (params: {|
-    root: boolean, path: string, title: string, content: string, thumbnail: string,
-    parentPath: string, order: number, availableLanguages: Map<string, string>, lastUpdate: Moment, hash: string
-  |}) {
-    const { order, parentPath, root, ...other } = params
+  constructor (params: {
+    root: boolean; path: string; title: string; content: string; thumbnail: string;
+    parentPath: string; order: number; availableLanguages: Map<string, string>; lastUpdate: Moment; hash: string;
```

Comment:
for those big objects you should consider creating a type like CategoryType, CityType, ... whatever